src/python_testing/TC_EWATERHTR_2_2.py

```diff
@@ -239,15 +267,22 @@ async def test_TC_EWATERHTR_2_2(self):
         time.sleep(6)
 
         self.step("9a")
+        event_data = events_callback.wait_for_event_report(Clusters.WaterHeaterManagement.Events.BoostEnded)
```

Comment:
Does this bomb out if it doesn't get it? i.e. do we need an assert here to validate it arrived or does it just throw an exception?

Reply:
It should throw an exception if the event does not arrive